docs/resources/Auto_Moderation.md

```diff
@@ -0,0 +1,98 @@
+# Auto Moderation
```

Comment:
even though this is a per guild resource I think it'd be good to separate this out into its own resources page like we did for Guild Scheduled Events for clarity, so I moved what was in Guilds Resource to here

Reply:
Perfect! Yes, it's a large topic.